cmd/capabilities/dispatcher/dispatcher.go

```diff
@@ -0,0 +1,169 @@
+// Sample run-helloworld is a minimal Cloud Run service.
+package main
+
+import (
+        "log"
+        "time"
+        "os"
+        "context"
+        "runtime"
+        "encoding/json"
+        "cloud.google.com/go/pubsub"
+        "github.com/apigee/registry/rpc"
+	    "github.com/golang/protobuf/jsonpb"
+        "google.golang.org/grpc/codes"
+        "google.golang.org/grpc/status"
+        "github.com/apigee/registry/cmd/capabilities/utils"
+        cloudtasks "cloud.google.com/go/cloudtasks/apiv2"
+        taskspb "google.golang.org/genproto/googleapis/cloud/tasks/v2"
+)
+
+// Make this configurable
+const topicName = "changes"
+const subscriptionName = topicName + "-pull-subscriber"
+
+func main() {
+    log.Print("Starting subscriber...")
+    ctx := context.Background()
+
+    // Setup and start the dispatcher server
+    dispatcher := &Dispatcher{}
+    err := dispatcher.setUp(ctx)
+    if err != nil {
+        log.Printf(err.Error())
```

Comment:
I am not using Fatal() here because it logs and then exits with exit code 1. The intention for not using Fatal was. the program should not exit but instead fail and return gracefully with appropriate Ack() or Nack() call.